datadog.conf.example

```diff
@@ -141,6 +141,15 @@ gce_updated_hostname: yes
 # Enable JMX checks for service discovery
 # sd_jmx_enable: no
 #
+# Docker labels as tags
+# We can extract docker labels and add them as tags to all metrics reported by service discovery.
+# All you have to do is supply a comma-separated list of label names to extract from containers when found.
+#
```

Comment:
I'd add a reference to docker_daemon's `collect_labels_as_tags` for docker/kube metrics